feature/login/src/main/java/com/yapp/feature/login/LoginScreen.kt

```diff
@@ -63,89 +47,40 @@ internal fun LoginRouteScreen(
     ) {
         LoginContent(
             loginState = loginState,
-            viewModel = viewModel
+            onSignUpClick = {viewModel.store.onIntent(LoginIntent.ClickSignUpButton)},
+            onLoginClick = {viewModel.store.onIntent(LoginIntent.ClickLoginButton)},
+            onIDChange = { viewModel.store.onIntent(LoginIntent.InputID(it)) },
+            onPWChange = {viewModel.store.onIntent(LoginIntent.InputPW(it))}
         )
     }
 }
 
-
 @Composable
 fun LoginContent(
-    loginState : LoginState,
-    viewModel : LoginViewModel
-) {
-    when(loginState){
-        LoginState.Loading -> LoginLoading()
-        LoginState.LoginError -> {}
-        LoginState.LoginSuccess -> {
-            LoginScreen(loginState,viewModel)
-        }
-    }
-}
-
-
-@Stable
-@Composable
-fun LoginScreen(
-    loginState : LoginState,
-    viewModel : LoginViewModel
+    loginState: LoginState,
+    onSignUpClick: () -> Unit,
+    onLoginClick : () -> Unit,
+    onIDChange: (String) -> Unit,
+    onPWChange : (String) -> Unit,
```

Comment:
```suggestion
    uiState: LoginState,
    onIntent: (Intent) -> Unit,
```

C: 요렇게 바꿔주시면 좋습니다 .. ㅎㅎ